source/_data/glossary.yml

```
@@ -286,15 +286,15 @@
 
 - term: Reload
   definition: >-
-    Updates Home Assistant configuration files. Changes are normally
-    automatically updated. However, when changes are made outside of the UI
-    and at a file level, Home Assistant is not aware of these changes and
-    requires the configuration file(s) to be reloaded to pick up any changes
-    made by going to **Settings** > **System** > **Restart Home Assistant**
-    (top right) > **Quick reload**. More granular reload options are available
-    in *YAML configuration reloading* section in **Developer tools** > **YAML**.
+    Refreshes Home Assistant configuration files. Changes are normally
```

```suggestion
    Applies the changes made in the Home Assistant configuration files. Changes are normally
```
Rephrased this a bit, because nothing happens to the files itself, I think.